spec/models/hmcts_common_platform/court_application_spec.rb

```diff
@@ -100,6 +100,12 @@
     end
   end
 
+  describe "respondents" do
+    it "are HmctsCommonPlatform::CourtApplicationParty objects" do
+      expect(court_application.respondents).to all(be_a(HmctsCommonPlatform::CourtApplicationParty))
+    end
+  end
+
```

Comment:
No tests for `judicial_result_ids` or `respondent_ids`?

Reply:
Or `court_application_type` ?

Is this one only used locally? Worth making it private?

Reply:
yeah good point, worth making `court_application_type` private.

Reply:
I'm going to remove `judicial_result_ids` and `respondent_ids` from this PR, it's outside of the scope. I'd also like to see if I can keep these out of our objects, because they're required only by the JSON API Serializer gem.